tsconfig.json

```
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+      "target": "es6",
```

I'd suggest setting this to `es2021`. Among other things it uses native node promises instead of a polyfill.
```suggestion
      "target": "es2021",
```